extensions/blocks/conversation/components/participants-controls.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * WordPress dependencies
+ */
+import { DropdownMenu, TextControl, Button } from '@wordpress/components';
+import { __, _x } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+
+function AddParticipantButton( { className, onAdd, participants = [] } ) {
+	return (
+		<div className={ `${ className }__participant` }>
+			<Button
+				className={ `${ className }__add-button` }
+				label={ __( 'Add Participant', 'jetpack' ) }
+				onClick={ () => onAdd( `Participant ${ participants.length + 1 }` ) }
+				isSecondary
+				isSmall
+			>
+				{ __( 'Add participant', 'jetpack' ) }
+			</Button>
+		</div>
+	);
+}
+
+function ParticipantsLabelControl( { className, participants, onChange, onDelete } ) {
+	return (
+		<div className={ `${ className }__participant-control` }>
+			{ participants.map( ( { participant, participantSlug } ) => (
```

Comment:
This triggers an error in the editor for me (WP 5.6, no Gutenberg):

`Uncaught TypeError: Cannot read property 'map' of undefined`

Reply:
I saw this bug too, but I thought I fixed it. Let me confirm.

Reply:
fixed here https://github.com/Automattic/jetpack/pull/17916/commits/99dc3b725145f948352d579b331a9d88ddbf0bb1